File: app/src/main/java/com/datarockets/mnchkn/fragments/charts/ChartsView.java
package com.datarockets.mnchkn.fragments.charts;

import com.datarockets.mnchkn.models.Player;

import java.util.ArrayList;

import lecho.lib.hellocharts.model.LineChartData;

public interface ChartsView {
    void drawChart(LineChartData lineChartData);
    void showPlayersList(ArrayList<Player> players);
}

File: app/src/main/kotlin/com/datarockets/mnchkn/ui/charts/ChartsView.kt
package com.datarockets.mnchkn.ui.charts

import com.datarockets.mnchkn.data.model.Player
import com.datarockets.mnchkn.ui.base.BaseView

import java.util.ArrayList

interface ChartsView : BaseView {
    fun showPlayersList(players: ArrayList<Player>)
}
